guides/micronaut-database-authentication-provider/micronaut-database-authentication-provider.adoc

```diff
@@ -23,6 +23,10 @@ If you use Micronaut Launch, select Micronaut Application as application type an
 
 include::{commondir}/common-default-package.adoc[]
 
+NOTE: If you have an existing Micronaut application and want to add the functionality described here, you can
+diffLink:[featureExcludes=spring-security-crypto]
+and apply those changes to your application.
+
```

Comment:
Looks good. The only improvement I'd add would be moving everything to the macro so we can add it easily to all the guides doing just `diffLink:[featureExcludes=spring-security-crypto]` and that generates the whole copy:

```
NOTE: If you have an existing Micronaut application and want to add the functionality described here, you can
https://micronaut.io/launch?features=security-jwt&features=security-ldap&features=hibernate-gorm&lang=GROOVY&build=GRADLE&test=SPOCK&name=micronautguide&type=DEFAULT&package=example.micronaut&activity=diff[view the dependency and configuration changes from the specified features, window="_blank"]
and apply those changes to your application.
```